intg-denonavr/driver.py

```diff
@@ -32,18 +32,22 @@
 
 async def receiver_status_poller(interval: float = 10.0) -> None:
     """Receiver data poller."""
+    # TODO: is it important to delay the first call?
```

Comment:
This was likely done to give the integration enough time to connect to the receiver at startup, since it's called from main().
But it's more of a cosmetic thing, it has to work both ways.